Scripts/Set-VMHostSecureNTP.ps1

```diff
@@ -0,0 +1,316 @@
+function Set-VMHostSecureNTP {
```

Comment:
I would suggest to tweak the interface of this on a little bit

Set-VMHostNtpServer [-VMHost] <vmhost> [-Ntp <ntp server>] [-Secure]

When `Set-VMHostNtpServer [-VMHost] <vmhost> -Ntp <ntp server>` is called just set the NTP Servers, btw you can utilize existing PowerCLI cmdlets  Remove-VMHostNtpServer and Add-VMHostNtpServer 

When `Set-VMHostNtpServer [-VMHost] <vmhost> -Secure` is called just set the firewall rules of configured Ntp Servers

When `Set-VMHostNtpServer [-VMHost] <vmhost> -Ntp <ntp server> -Secure` is called replace ntp servers wit the desired ones and configre the firewall rules

Reply:
The Secure inline function does currently only set the existing NTP Servers and keeps sere service config as it is.

Only for SetSecure the NTP Servers are currently required, and also the NTP Service (enabled) is configured not only the NTP Servers. 

I understand you suggestion, but I would prefer the current behavior.

Reply:
Ok, it was just a suggestion, you are the author 👍